system/emergency_handler/README.md

```diff
@@ -8,7 +8,7 @@ Emergency Handler is a node to select proper MRM from from system failure state
 
 ### State Transitions
 
-![fail-safe-state](https://tier4.github.io/autoware.proj/tree/main/design/apis/image/fail-safe-state.drawio.svg)
+![fail-safe-state](https://github.com/tier4/AutowareArchitectureProposal.proj/blob/main/docs/design/apis/image/fail-safe-state.drawio.svg)
```

Comment:
Ah, I meant to copy this image to this repository!

Reply:
But considering the new design of APIs will be prepared in the future, it might be okay.
So I'll approve this.

Reply:
Oops.